src/compiler/CompiledResponseHeader.ts

```diff
@@ -20,14 +20,16 @@ export default class CompiledResponseHeader {
       if (this.ignoreHeaders.includes(name)) {
         continue;
       }
+
+      const headerNameLower = name.toLowerCase();
```

Comment:
Since we now have all the keys in the schema as lower case, what's making sure that the value comes in to `validate()` function has all the key converted to lower case as well?

Reply:
@supertong Excellent catch! I forgot that for an incoming request, node.js is already lowercasing the header name. For the response, the code in this library needs to do it before calling validate(). I just committed some changes. Thanks